src/app/containers/Bulletin/index.stories.jsx

```diff
@@ -33,33 +35,50 @@ const audioFixture = bulletinFixture('RadioBulletin');
 const liveTvFixture = liveBulletinFixture('TVBulletin');
 const audioLiveFixture = liveBulletinFixture('RadioBulletin');
 
-const getBulletinPromo = platform => item => (
-  <ServiceContextProvider service="news">
+const getBulletinPromo = (platform, service, item) => (
+  <ServiceContextProvider service={service}>
     <RequestContextProvider
```

Comment:
Everything looks good, translations match the spreadsheet. The only thing is that storybook gives a warning for Bulletin : `Warning: Failed prop type: The prop pathname is marked as required in RequestContextProvider, but its value is undefined`.

Reply:
Added also to the `StoryPromo` stories as it was also missing and logging a warning.